streams/src/test/java/org/apache/kafka/streams/state/internals/SessionKeySchemaTest.java

```diff
@@ -63,6 +63,25 @@ public void shouldFetchExactKeySkippingShorterKeys() throws Exception {
         assertThat(results, equalTo(Arrays.asList(1, 5)));
     }
 
+    @Test
+    public void boundsSanity() throws Exception {
+        // easy
+        Bytes upper = sessionKeySchema.upperRange(Bytes.wrap(new byte[]{1}), Long.MAX_VALUE);
```

Comment:
would prefer to see separate tests for each of these. Describing what they are testing